src/node/desktop/src/main/detect-r.ts

```diff
@@ -113,6 +112,7 @@ export async function promptUserForR(platform = process.platform): Promise<Expec
     // save the stored version of R
     const path = data.binaryPath as string;
     ElectronDesktopOptions().setRBinDir(dirname(path));
+    ElectronDesktopOptions().setRstudioPath(path);
```

Comment:
This is the path to R, not RStudio. Shouldn't this be something like `setRBinaryExecutablePath()`?

In addition, if we're setting this, it seems like `rBinDir` is redundant, or at least the executable path could be computed from the binary path. In other words, I don't see why this is necessary.

Reply:
I followed the naming used by `rBinDir` as `rstudioPath` was also declared there and it worked. Not sure how `rBinDir` is being used right now. That's why I created that new one for saving the exact R binary executable path so we could compare on the Choose R Modal to correctly check the current set option. Just let me know if you think I should rename something or do it differently so I can deploy a change!

Reply:
Perhaps we should replace the `RBinDir` preference and replace it with the `RExecutable` preference, and compute the `RBinDir` from that preference where appropriate. It's easier to compute the binary directory from the executable path anyhow, I think.